### DISCUSSION TITLE
ProHD internal tuner remote?

### DISCUSSION THREAD

**Discussion by RandomMan719** [GerryDazoo/Slinger#192] [Q&A] [answered]
I was wondering if there is any update, on getting the internal tuner remote working for ProHD's?  I have quite a few of them, and use the internal tuner on all of them.  Thank you, for your work on this project.

**Comment by GerryDazoo** [comment, marked as answer]
Coming soon. Email me at gerry@dazzo.net if you want a beta version.

**Reply by RandomMan719** [reply]
I am getting a bounce back message when I try to email you.  It says "no mx record found for domain=dazzo.net"

**Reply by GerryDazoo** [reply]
opps. dazoo

**Reply by RandomMan719** [reply]
The beta is working for me. I can change channels on the internal tuner's of the ProHDs.  Thank you.
